code/modules/vehicles/mecha/combat/marauder.dm

```diff
@@ -18,9 +18,9 @@
 	max_equip_by_category = list(
 		MECHA_L_ARM = 1,
 		MECHA_R_ARM = 1,
-		MECHA_UTILITY = 5,
+		MECHA_UTILITY = 3,
```

Comment:
It's probably not worth adjusting values on the Marauder. The only time crew encounter these mechs is via admin spawn, and I don't think anyone cares to balance pass admin-spawned mechs unless they happen to want to adjust the Mauler, the nuclear operative version, which is better done on that subtype specifically.

Reply:
I've got no beefs about the marauder or mauler (or the reticence, really, for as much as it shows up - it just basically has Durand armor values and a slot whereas the HONK has negative melee armor and no slots.)

The Durand, Gygax, SI, and to a lesser extent the Phazon (its armor should be its phasing, not stacking modules, but it should also have more utility for tricks and crafty plays) are my main targets here.

Reply:
well-taken, just going to stick to the ones that really matter and appear here